tx.go

```diff
@@ -152,24 +155,39 @@ func (tx *Tx) RootPage() (*Page, error) {
 	if tx.rootID < 2 {
 		return nil, nil
 	}
-	return tx.Page(tx.rootID)
+	return tx.getPage("txfile/tx-access-root", tx.rootID)
```

Comment:
Should this use a const?

Reply:
👍  yeah, I got a little lazy at times...